script.js

```diff
@@ -136,8 +162,32 @@ function projectPopup(position) {
             </div>
           </div>
         </div>
-    `);
+    `;
   popup.classList.add('popup');
   portfolios.appendChild(popup);
 }
+const projectnBtn = document.querySelectorAll('.see-project');
+projectnBtn.forEach((btn, ind) => {
+  btn.addEventListener('click', () => {
+    projectPopup(ind);
+    const popup = document.querySelector('.popup');
+    document.querySelector('.close-icon').addEventListener('click', () => {
+      portfolios.removeChild(popup);
+    });
+  });
+});
 
+// ######################## Contact Form Validation Section #######################
+const validateEmail = (email) => {
+  if (email !== email.toLowerCase()) return false;
+  return true;
+};
+
+contactForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (!validateEmail(contactForm.elements.email.value.trim())) {
+    alert('Email should be all lower case only');
```

Comment:
Great job team 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md), you are supposed to print the error message next to the submit button. This is not the case for your design. You may try not to use the alert function to render the error message.
![alert](https://user-images.githubusercontent.com/63917725/180440100-1e7c9a5f-0f01-4cdf-9554-01b83cc84fbe.PNG)